### DISCUSSION TITLE
How to turn off auto increment for primary key

### DISCUSSION THREAD

**Discussion by dpacmittal** [mikro-orm/mikro-orm#2175] [Q&A] [answered]
I have a table which stores github installation ids which are provided by github api and are unique. I want this column to be the primary key but I don't want it to auto increment. How do I turn off auto increment for this column?

**Comment by B4nan** [comment, marked as answer]
In v5 there is [an option](https://mikro-orm.io/docs/next/api/modules/core#propertyoptions) for this, in v4 just change the schema manually, it should be ignored in schema diffing.

Or keep it defined as autoincrement, it should not really matter, you can still provide the PK values even for autoincrement column.

**Comment by dpacmittal** [comment]
Thankyou. I'll keep the column as auto increment and pass in the id manually while storing, that sounds good.
